Pear/Controller/CommunityViewController.swift

```diff
@@ -64,20 +64,12 @@ class CommunityViewController: UIViewController {
     }
 
     private func getUsers() {
-        NetworkManager.shared.getUsers().observe { response in
+        Networking2.getAllUsers { [weak self] allUsers in
```

Comment:
you should be able to do your custom paginated get all users here now along with the search query